src/SDKs/DataLake.Analytics/DataLakeAnalytics.Tests/ScenarioTests/JobOperationTests.cs

```diff
@@ -175,10 +175,10 @@ public void SubmitGetListCancelTest()
                 Assert.Contains("E_CSC_USER_SYNTAXERROR", ((USqlJobProperties)compileResponse.Properties).Diagnostics[0].Message);
 
                 // list the jobs both with a hand crafted query string and using the parameters
-                listJobResponse = clientToUse.Job.List(commonData.SecondDataLakeAnalyticsAccountName, select:  "jobId" );
-                Assert.NotNull(listJobResponse);
+                // listJobResponse = clientToUse.Job.List(commonData.SecondDataLakeAnalyticsAccountName, select:  "jobId" );
```

Comment:
I would put a note in here indicating the issue in a TODO comment since this functionality should be re-enabled.